src/main/java/me/ruibin/leto/tasklist/InvalidTaskException.java

```diff
@@ -0,0 +1,15 @@
+package me.ruibin.leto.tasklist;
+
+import me.ruibin.leto.ui.Ui;
+
+/** Exception for invalid input for a Task */
+public class InvalidTaskException extends Exception {
```

Comment:
It seems like almost all the exceptions extend InvalidTaskException; even ones like InvalidCmdException.
To me its not intuitive that an invalid command constitutes an invalid task.
Perhaps the naming/inheritance could be modified to make it more intuitive.
Otherwise, I think it makes sense!